src/editor/components/Setting/ComponentAttr/FormElementRenderer.tsx

```
@@ -0,0 +1,107 @@
+  Switch,
+  Radio,
+  Checkbox,
+  Segmented,
+} from "antd";
+import type { ComponentSetter } from "../../../stores/component-config";
+import JsonEditor from "../../common/JsonEditor";
+import { BreadcrumbSetter } from "./BreadcrumbSetter";
+import AttrListSetter from "../../common/AttrListSetter";
+import { normalizeOptions } from "../../../utils/formUtils";
+
+type AnyObject = Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)